src/interactive/promptmodal.py

```diff
@@ -0,0 +1,11 @@
+import discord
+
+
+class PromptModal(discord.ui.Modal, title="e-bot"):
+    response = discord.ui.TextInput(label="Answer")
+
+    async def on_submit(self, interaction: discord.Interaction):
+        # pylint: disable=arguments-differ,attribute-defined-outside-init
+        self.stop()
+        self.interaction = interaction
+        self.answer = self.response.value
```

Comment:
This seems only to be used in `lashprompt.py`? Could we just stick it at the top of that file as a utility class, or is the expectation that this will grow/be reused?

Reply:
I thought there's a chance it may be reused in other things. And it's fairly generic anyway.